docs/post/memcards.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Memory Cards"
```

Comment:
Might be a good idea to set the title (and the name of the file) to something indicating it is specifically for MyMC? As it is, it suggests it is more a general piece on memory cards.